source/MRViewer/MRSurfaceManipulationWidget.cpp

```diff
@@ -74,6 +74,9 @@ void SurfaceManipulationWidget::reset()
 
 void SurfaceManipulationWidget::setSettings( const Settings& settings )
 {
+    if ( mousePressed_ )
+        return;
```

Comment:
why add this?

Reply:
it's problem if settings change during edit mesh in progress